app/routes/__private/user/employees.$uuid.tsx

```diff
@@ -56,15 +75,12 @@ export const loader: LoaderFunction = (args) => {
 
 export const action: ActionFunction = (args) => {
   return remixAppAction(args, {
-    DELETE: ({ userId, params, requestId, ...rest }) => {
-      console.log("params", params);
-      console.log("rest", rest);
-      return fireUserEmployee({
+    DELETE: ({ userId, params, requestId }) =>
+      fireUserEmployee({
```

Comment:
need to still delete the key here, and add a history item record for the employee